mote/__init__.py

```diff
@@ -29,7 +29,7 @@
 
 import flask, random, string, json, util, re
 import dateutil.parser, requests, collections
-import logging
+import logging, soke
```

Comment:
The import is used elsewhere in the file, but it's never caused problems even though it was never imported before. 

pyflakes was complaining about the missing import, so I added it. I'm not sure why it was working before even though the import was missing. Maybe you have a better idea of why?